packages/core/src/interactable-card/InteractableCard.tsx

```diff
@@ -0,0 +1,178 @@
+
+export interface InteractableCardProps extends ComponentPropsWithoutRef<"div"> {
+  /**
+   * Accent border position: defaults to "bottom"
+   */
+  accent?: "bottom" | "top" | "left" | "right";
+  /**
+   * **Deprecated:** Use the `accent` prop instead
+   *
+   */
+  accentPlacement?: "bottom" | "top" | "left" | "right";
+  /**
+   * If `true`, the card will be disabled.
+   */
+  disabled?: boolean;
+  /**
+   * If `true`, the card will have selected styling.
+   */
+  selected?: boolean;
+  /**
+   * Callback fired when the selection changes.
+   * @param event
+   */
+  onChange?: (event: SyntheticEvent<HTMLDivElement>) => void;
+  /**
+   * Styling variant; defaults to "primary".
+   */
+  variant?: "primary" | "secondary";
+  /**
+   * Value of card (for selectable use case).
+   */
+  value?: InteractableCardValue;
+}
+
+export const InteractableCard = forwardRef<
+  HTMLDivElement,
+  InteractableCardProps
+>(function InteractableCard(props, ref) {
+  const {
+    accent,
+    accentPlacement,
+    children,
+    className,
+    disabled: disabledProp,
+    selected: selectedProp,
+    variant = "primary",
+    value,
+    onBlur,
+    onChange,
+    onClick,
+    onKeyUp,
+    onKeyDown,
+    ...rest
+  } = props;
+
+  const targetWindow = useWindow();
+  useComponentCssInjection({
+    testId: "salt-interactable-card",
+    css: interactableCardCss,
+    window: targetWindow,
+  });
+
+  const interactableCardGroup = useInteractableCardGroup();
+
+  const interactableCardGroupSelected = interactableCardGroup
+    ? interactableCardGroup.isSelected(value)
+    : selectedProp;
+
+  const disabled = interactableCardGroup?.disabled || disabledProp;
+
+  const [selected, setSelected] = useControlled({
+    controlled: interactableCardGroupSelected,
+    default: Boolean(selectedProp),
+    name: "InteractableCard",
+    state: "selected",
+  });
+
+  const role = interactableCardGroup
+    ? interactableCardGroup.multiSelect
+      ? "checkbox"
+      : "radio"
+    : "button";
+
+  const isMultiselect =
+    interactableCardGroup && interactableCardGroup.multiSelect;
+
+  const isFirstChild =
+    interactableCardGroup && interactableCardGroup.isFirstChild(value);
+
+  const ariaChecked =
+    role === "radio" || role === "checkbox" ? selected : undefined;
+
+  const handleClick = (event: MouseEvent<HTMLDivElement>) => {
+    if (interactableCardGroup && !disabled) {
+      interactableCardGroup.select(event, value);
+      setSelected(!selected);
+    }
+    onChange?.(event);
+    onClick?.(event);
+  };
+
+  let tabIndex;
+
+  if (interactableCardGroup) {
+    if (disabled) {
+      tabIndex = -1;
+    } else if (isMultiselect) {
+      tabIndex = 0; // All items focusable in multi-select
+    } else {
+      // Single select: Only selected or first item (if none are selected) is focusable
+      tabIndex = selected ? 0 : -1;
+      if (!interactableCardGroup?.value && isFirstChild) {
```

Comment:
nit: question mark shouldnt be needed here (only highlighting bc it makes the logic a little simpler to understand if its gone - ie checking if theres a value for the group or not)